pkg/config/config_test.go

```diff
@@ -542,12 +543,18 @@ func (s *MySuite) TestSetCLIVariables(c *C) {
 	cliRegion := "cli_region"
 	cliZone := "cli_zone"
 	cliKeyVal := "key=val"
+	cliKeyBool := "true"
```

Comment:
I'd like to see the full set of requirements from the design being tested here, at least in a simple form:

> Parse simple variables provided under a --vars flag
> - Boolean: true
> - Integers: 3
> - Floats: 3.14
> - Strings: “foo”
>
> Parse complex variables provided under a --vars flag
> - Array: [1, 2, 3]
> - Map: {a: 1, b: 2}
> - Array with maps: [foo, {a: 1, b: 2}]
> - Map with arrays: {foo: [1, 2, 3], bar: [a, b, c]}

Reply:
Done.